source/transactions-convenient-api/tests/commit-writeconcernerror.yml

```diff
@@ -35,69 +35,9 @@ tests:
                   insertedId: 1
         result:
           errorCodeName: WriteConcernFailed
-          # Per transactions spec, drivers add UnknownTransactionCommitResult
-          # label for WriteConcernFailed errors; however, neither the driver
-          # nor withTransaction() will retry the commit in this case.
           errorLabelsContain: ["UnknownTransactionCommitResult"]
           errorLabelsOmit: ["TransientTransactionError"]
-    expectations:
-      -
-        command_started_event:
-          command:
-            insert: *collection_name
-            documents:
-              - { _id: 1 }
-            ordered: true
-            lsid: session0
-            txnNumber: { $numberLong: "1" }
-            startTransaction: true
-            autocommit: false
-            # omitted fields
-            readConcern: ~
-            writeConcern: ~
-          command_name: insert
-          database_name: *database_name
-      -
-        command_started_event:
-          command:
-            commitTransaction: 1
-            lsid: session0
-            txnNumber: { $numberLong: "1" }
-            autocommit: false
-            # omitted fields
-            readConcern: ~
-            startTransaction: ~
-            writeConcern: ~
-          command_name: commitTransaction
-          database_name: admin
-    # The write operation is still applied despite the write concern error
-    outcome: &outcome
-      collection:
-        data:
-          - { _id: 1 }
```

Comment:
I'm not sure how this was passing, unless:

 * The server was actually committing the transaction despite our fail point configuration, and the data was present by the time drivers inspected the outcome collection.
 * Drivers that were running these tests had some error in their test runners, such as not asserting the outcome collection at all if the operation experienced an error

Something to look into either way. I suggest any drivers with working test runners verify that the `outcome` block is still processed regardless of whether the operation(s) succeeds or fails.

Reply:
CC @jstewart-mongo.  Can you check this please?

Reply:
I have confirmed in our tests that the `outcome` block is processed regardless of whether the operation(s) succeed or fail.